tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1281,6 +1281,84 @@ final class TestingServer(
     Assertions.assertNoDiff(obtained, expected)
   }
 
+  def assertDefinitionAtLocation(
```

Comment:
If these methods are very specific to the Ammonite suite I would move them there.  We already have things like `assertHover` here, so I would like to avoid duplicating the ways of asserting specific things.

We do not have assert definition in TestingServer, but I would go a different way showing the whole file in the test case like shown in the other comment.

Reply:
Just moved that to `BaseAmmoniteSuite`.